coremltools/converters/mil/frontend/torch/torchir_passes.py

```diff
@@ -97,7 +97,7 @@ def _construct_nodes_to_fuse_inputs(nodes_to_fuse):
             input_name = node.inputs[idx]
             node.inputs[idx] = _get_updated_name(input_name, updated_tensor_count)
 
-        if node.kind in ("select", "slice"):
+        if node.kind in ("empty", "select", "slice"):
```

Comment:
I am surprised why `empty` falls in this group.
Can you show a simple pattern that the `empty` is include?

Reply:
Here are the relevant lines from the PyTorch graph, in #1447, after we lower it:
```
  %mask.1 : Float(1, 2, 2, strides=[4, 2, 1], requires_grad=0, device=cpu) = aten::empty(%7549, %7525, %7524, %7523, %7527, %7524), scope: __module.text_model # /Users/tobyroseman/miniconda3/envs/test/lib/python3.9/site-packages/transformers/models/clip/modeling_clip.py:673:0
  %mask.3 : Float(1, 2, 2, strides=[4, 2, 1], requires_grad=0, device=cpu) = aten::fill_(%mask.1, %7522), scope: __module.text_model # /Users/tobyroseman/miniconda3/envs/test/li
```

I agree it seems strange/wrong for the input of a fill to be uninitialized memory.